pages/api/topics.js

```diff
@@ -1,5 +1,32 @@
-import { topics } from "../api/data";
+import fs from "fs";
+import path from "path";
 
-export default function handler(req, res) {
-  res.status(200).json(topics);
+//
+// Fetch ALL topics from local Data
+//
+
+export function getLocalData() {
+  const fullPath = path.join(process.cwd(), "pages", "api", "data.json");
+  const fileContents = fs.readFileSync(fullPath, "utf8");
+  const localData = JSON.parse(fileContents).topics;
+
+  return { localData };
+}
+
+//
+// Fetch ALL topics from API
+//
+
+export async function getTopics(language) {
+  //
+  console.log(
+    `api url is: ${process.env.NEXTJS_CONTENT_API}  language is: ${language}`
```

Comment:
The language here is an object, so it looks like this on my command line: 
<img width="261" alt="Screen Shot 2021-06-01 at 13 19 54" src="https://user-images.githubusercontent.com/2454380/120365705-c940e880-c2dc-11eb-84be-4d6d0bffcba7.png">

When I print out the language variable by itself, I see this: `{ locales: [ 'en', 'fr' ], locale: 'en', defaultLocale: 'en' }`.

So probably we can put `language is: ${language.locale}`.

Reply:
Fixed this.

Reply:
actually I forgot to remove this console.log.  but yes we do need to change the language to locale almost everywhere